.circleci/config.yml

```diff
@@ -110,6 +110,21 @@ jobs:
       - *restore_yarn_offline_mirror
       - *restore_yarn_cache
       - *install_js
+      - run:
+          name: material-ui-styles
+          command: |
+            # latest commit
+            LATEST_COMMIT=$(git rev-parse HEAD)
+
+            # latest commit where packages/material-ui-styles was changed
+            FOLDER_COMMIT=$(git log -1 --format=format:%H --full-diff packages/material-ui-styles)
+
+            if [ $FOLDER_COMMIT = $LATEST_COMMIT ]; then
+              echo "changes, let's run the tests"
```

Comment:
We need to rethink this strategy. Currently we assume that every package works in isolation. But the core now relies on  styles and lab always relied on core. I know that jest has some "only run tests on files that were affected by changes" but I don't know if this works with monorepos and I believe that CI should always run the full test suite.

Reply:
I'm removing this `"run"`. I agree, better run the styles test suite every single time.